src/navigate.js

```diff
@@ -66,6 +66,7 @@ class ViewSlot {
 		for (let x of items) {
 			let a = document.createElement('a')
 			a.href = x.href
+			a.target = x['target'] ?? '_self'
```

Comment:
- should just be x.target || '_self' (i don't use ?? anywhere else in the code (..i hope i had a good reason for this))

Reply:
fineee